schema_editor/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/azavea/static-core:0.1.0
+FROM node:4-slim
```

Comment:
Once this is merged I think we can let ops know that they can remove this image from Quay; I assume it's not being used anywhere else.